src/components/stx-avatar.tsx

```diff
@@ -4,8 +4,8 @@ import { BoxProps, Circle, color, DynamicColorCircle } from '@stacks/ui';
 
 export const StxAvatar: React.FC<BoxProps> = props => {
   return (
-    <Circle backgroundColor={color('invert')} {...props}>
-      <StxIcon size="12px" />
+    <Circle backgroundColor="#5347F5" {...props}>
```

Comment:
```suggestion
    <Circle backgroundColor={color('accent')} {...props}>
```

Reply:
Got it, thanks, will change. Didn't see that color was captured in a string. 👍